README.md

```diff
@@ -50,36 +50,24 @@ pluginManagement {
 }
 ```
 
-`build.gradle`
+`build.gradle.kts`
 
-```groovy
+```kotlin
 plugins {
-    id("org.jetbrains.kotlin.jvm") version "1.9.0"
-    id("com.google.devtools.ksp") version "1.9.0-1.0.13"
+    id("org.jetbrains.kotlin.jvm")
+    id("com.google.devtools.ksp")
```

Comment:
I thought that using specific versions might give the impression that kotlin-inject will only work with those versions

Reply:
Hm, fair enough. It's so you can copy those lines and have it actually work, I tend to update these to whatever that latest is at the time of each release but you are right that it might be giving a false impression.

Reply:
In some projects specifying the versions can cause an error 😅

In general my approach with these things is that the README for libraries don't need to show how to set up the whole project.